src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -1601,7 +1601,7 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "NewVersionFeature_215_ticketValidation_title" = "Zertifikatprüfung bei Buchungen";
 
-"NewVersionFeature_215_ticketValidation_description" = "Bereits bei einer Ticketbuchung können Sie den Nachweis erbringen, dass Sie geimpft, genesen oder negativ getestet sind. Dazu scannen Sie einen vom Buchungssystem angezeigten QR-Code und wählen dann in der App das passende Zertifikat aus. Wird Ihr Zertifikat erfolgreich geprüft und erfüllt die Anforderungen, wird es an das Buchungssystem übertragen.";
+"NewVersionFeature_215_ticketValidation_description" = "Bereits bei einer Ticketbuchung können Sie den Nachweis erbringen, dass Sie geimpft, genesen oder negativ getestet sind. Dazu scannen Sie einen vom Buchungssystem angezeigten QR-Code und wählen dann in der App das passende Zertifikat aus. Das Ergebnis der Zertifikatsprüfung wird vom Prüfpartner an das Buchungssystem übertragen.";
```

Comment:
@KathrinLuecke What exactly is the "Prüfpartner" in this case? Is it the owner of the "Buchungssystem"? This is a bit confusing for normal users.

Reply:
@Ein-Tim No, it's not the Buchungssystem, it's a validation service provided by another company. Maybe "Prüfservice" would be better here? Or do you have other suggestions?

Reply:
@KathrinLuecke 

Yeah, "Prüfservice" makes it clearer but still this leads to the question who provides this "Prüfservice". 

Is this somewhere defined nearer (e.g. in the privacy note or somewhere where the user could search for it to understand this?)

Reply:
The process with the "Prüfpartner" is explained in the privacy policy: https://github.com/corona-warn-app/cwa-app-ios/blob/release/2.15.x/src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html#L775

I think it's also transparent in the process: Before anything is transmitted to the Prüfpartner, you have to give your consent and it's well described on the screen what happens: 

Übermittlung Ihres Zertifikats zur Prüfung
Das ausgewählte Zertifikat und Ihre Buchungsdaten werden nun an den Prüfpartner des Anbieters übermittelt.  Für die Übermittlung Ihrer Daten an den Prüfpartner ist Ihr Einverständnis erforderlich. 
Ihr Einverständnis 
Durch Antippen von „Einverstanden“ willigen Sie in die Übermittlung der folgenden Daten an den Prüfpartner ein: 
Zertifikatsinhalt und elektronische Signatur des ausgewählten Zertifikats (u. a. Name und Geburtsdatum des Zertifikatsinhabers, Gültigkeitsdauer, Aussteller und Ausstellungsdatum, Angaben zu Impfstoff/Test/Genesung). 
Die bei dem Anbieter abgerufenen Buchungsdaten:
Zeitraum, in dem das Zertifikat gültig sein muss (Reisedauer oder Veranstaltungsdatum)

bei Reisebuchungen: Ausgangsland und Zielland der Reise

bei grenzüberschreitenden Reisen: Anforderungen des Ziellandes an das Zertifikat

weitere Anforderungen des Anbieters an das Zertifikat (z. B. welche Impfstoffe oder Testverfahren akzeptiert werden)
Der Prüfpartner überprüft, ob die elektronische Signatur echt und das technische Ablaufdatum des Zertifikats noch nicht erreicht ist. Das technische Ablaufdatum und die übermittelten Zertifikatsinhalte können Sie in der App in der Detailansicht des Zertifikats einsehen.

Die Zertifikate enthalten sensible Gesundheitsdaten. Erlauben Sie daher nur solchen Anbietern die Überprüfung, die Ihnen bekannt sind und denen Sie vertrauen.

Für die Datenverarbeitung durch den Anbieter und den Prüfpartner ist das RKI nicht verantwortlich. Lesen Sie bitte deren   Datenschutzhinweise, damit Sie wissen, wozu und wie Ihre Daten verwendet werden.

Der Prüfpartner teilt dem Anbieter sofort das Prüfungsergebnis mit. Es wird nur mitgeteilt, ob die Prüfung erfolgreich war oder nicht.


I'm a bit reluctant to chang "Prüfpartner" to "Prüfservice", because Prüfpartner is already used at so many places (and for people without much technical knowledge, it might not make a difference anyhow.)

Reply:
@KathrinLuecke 

Okay yes, with this text this is fine! Thanks for your answers!